src/core/api/ble_secure_api.cpp

```diff
@@ -32,6 +32,7 @@
  */
 
 #include "openthread-core-config.h"
+#include <openthread/error.h>
```

Comment:
Do we need this `#include <openthread/error.h>` here in the `.cpp` file? Can we remove it?

I think we should not need to include `error.h` here. We include `#include "instance/instance.hpp"` below (which should cover everything).

Reply:
fixed.